pages/pr/index.html

```diff
@@ -0,0 +1,14 @@
+---
+layout: pr_layout
+permalink: /pr
```

Comment:
I think it's better to make is more descriptive. like: `/get-paid-for-pull-requests`